discrete_summary_checker/example/Check_Format.py

```diff
@@ -33,7 +33,7 @@
 # #### Summary Sheet
 # Load the summary sheet. Make sure to navigate to the correct directory and have the correct file name entered.
 
-summary_sheet = pd.read_excel("../data/Irminger_Sea-06_AR35-05_Discrete_Sample_Summary.xlsx")
+summary_sheet = pd.read_excel("/home/andrew/Documents/OOI-CGSN/QAQC_Sandbox/Ship_data/data/Papa-04/Station_Papa-04_RB1605_Discrete_Summary_2022-01-03_ACR.xlsx")
```

Comment:
Is this a path that is going to be accessible? Does the file need to be moved to ../data or some other accessible location?

Reply:
I updated it to a generic placeholder of "put in path to sample sheet"